app/uploaders/logo_uploader.rb

```diff
@@ -0,0 +1,49 @@
+class LogoUploader < BaseUploader
+  MAX_FILE_SIZE = 3 # Megabytes
+  EXTENSION_ALLOWLIST = %w[svg png jpg jpeg jpe].freeze
+  IMAGE_TYPE_ALLOWLIST = %i[svg png jpg jpeg jpe].freeze
+  CONTENT_TYPE_ALLOWLIST = %w[image/svg+xml image/png image/jpg image/jpeg].freeze
+
+  def store_dir
+    "uploads/logos/"
```

Comment:
Nitpick: methods in this class assign seem to return static values via constants, except for this one.

Reply:
✅